app/javascript/components/dropdowns/track-menu/ResetTrackModal.tsx

```diff
@@ -97,6 +97,19 @@ export const ResetTrackModal = ({
               {track.title}
             </li>
           </ul>
+          <p>
+            <strong>You will keep:</strong>
+          </p>
+          <ul>
+            <li>
+              The local version of any {track.title} solutions you wrote outside
+              Exercism
+            </li>
```

Comment:
I'm not sure this adds much value 🤷 @iHiD ?

Reply:
Not deleting local code seems obvious, but I figured it was worth clarifying because it mentions "delete all your solutions". Happy to pull though. I could imagine questions around "well, does the CLI know I've reset the track and wipe my local directories when next run?". That wouldn't happen of course, but that's why I added the clarification.